tikv/raftstore/engine.go

```diff
@@ -225,3 +235,117 @@ func (wb *WriteBatch) Reset() {
 	wb.safePointLock = 0
 	wb.safePointSize = 0
 }
+
+// Todo, the following code redundant to unistore/tikv/worker.go, just as a place holder now.
+
+const delRangeBatchSize = 4096
+
+const maxSystemTS = math.MaxUint64
+
+func deleteRange(db *DBBundle, startKey, endKey []byte) error {
+	// Delete keys first.
+	keys := make([][]byte, 0, delRangeBatchSize)
+	oldStartKey := mvcc.EncodeOldKey(startKey, maxSystemTS)
+	oldEndKey := mvcc.EncodeOldKey(endKey, maxSystemTS)
+	txn := db.db.NewTransaction(false)
+	reader := dbreader.NewDBReader(startKey, endKey, txn, 0)
+	keys = collectRangeKeys(reader.GetIter(), startKey, endKey, keys)
+	keys = collectRangeKeys(reader.GetIter(), oldStartKey, oldEndKey, keys)
+	reader.Close()
+	if err := deleteKeysInBatch(db, keys, delRangeBatchSize); err != nil {
+		return err
+	}
+
+	// Delete lock
+	lockIte := db.lockStore.NewIterator()
+	keys = keys[:0]
+	keys = collectLockOrRollbackRangeKeys(lockIte, startKey, endKey, keys)
+	keys = collectLockOrRollbackRangeKeys(lockIte, oldStartKey, oldEndKey, keys)
+	if err := deleteLocksInBatch(db, keys, delRangeBatchSize); err != nil {
+		return err
+	}
+
+	// Delete rollback
```

Comment:
We don't need to delete rollback, it will be GCed automatically.

Reply:
ok